testcafe/tests/helpers/utils.js

```diff
@@ -29,6 +29,23 @@ export const getPageUrl = ClientFunction(() => window.location.href)
 
 export const goBack = ClientFunction(() => window.history.back())
 
+export const getNavigatorOs = ClientFunction(() => navigator.platform)
+
+//User Agent is needed for VisualReview, but we don't need the all string
+export const getNavigatorName = ClientFunction(() => {
+  var aKeys = ['MSIE', 'Firefox', 'Safari', 'Chrome', 'Opera'],
```

Comment:
Please don't use `var` to declare variables, and we can use complete variable names, no need to shorten them.